src/exprestive.coffee

```diff
@@ -38,8 +38,8 @@ class Exprestive
     # Save reverse routes
     @reverseRoutes = @options.routes
 
-    # Set res.locals.routes if the routes option was not set
-    @middlewareRouter.use @setReverseRoutesOnReqLocals if @optionIsDefault 'routes'
+    # Set res.locals.routes
+    @middlewareRouter.use @setReverseRoutesOnReqLocals
```

Comment:
We can get rid of the `optionIsDefault` method now, this is the only place it was used